browser/psst/psst_ui_delegate_impl.cc

```diff
@@ -12,7 +12,8 @@ PsstUiDelegateImpl::~PsstUiDelegateImpl() = default;
 
 void PsstUiDelegateImpl::UpdateTasks(
     long progress,
-    const std::vector<PolicyTask>& applied_tasks) {
+    const std::vector<PolicyTask>& applied_tasks,
+    const mojom::PsstStatus status) {
```

Comment:
can we split out adding `PsstStatus` into a separate PR? It can go before or after this one, doesn't matter to me. This seems like a distinct change that isn't directly related to the time out